.github/workflows/linter.yml

```diff
@@ -2,12 +2,13 @@ name: Linter
 on:
   push:
     branches:
-      - v[0-9]+.[0-9]+.x-celestia
+      - master
     paths:
       - "**.md"
       - "**.yml"
       - "**.yaml"
   pull_request:
+  branches: [master]
```

Comment:
This line seems wrong. Is it trying to only make CI run on PR against `master`? If so then it should be indented.

In addition, if we're going to do that, then this file should just be removed entirely. Or the file should remain unchanged and this PR merged with intentionally failing linter and an issue filed to fix it.

Reply:
I think fixing it is a mistake since it would cause a large diff between us and upstream, so I deleted the file. Wasn't sure if we wanted to do this or not, as upstream seems to be happy having that check never run on this branch. https://github.com/celestiaorg/celestia-core/pull/792/commits/ec66d5daebf48f5858c54f5caeb18a48c7179a6b